app/components/post_body/post_body.js

```diff
@@ -71,6 +71,7 @@ export default class PostBody extends PureComponent {
         shouldRenderJumboEmoji: PropTypes.bool.isRequired,
         theme: PropTypes.object,
         location: PropTypes.string,
+        canPost: PropTypes.bool.isRequired,
```

Comment:
do you need this here?

Reply:
No, the only place where `<PostBody>` is used doesn't pass it in as a prop, and anyway the `PostOptions` screen looks it up. I can either: 

1) Remove this prop and don't pass it to to the `PostOptions` screen, or
2) Change it to make post_body/index.js lookup the `canPost` value in Redux, leave this prop here, and pass it all the way down to the `PostOptions` screen and remove the lookup there.

Preference?

Reply:
@mkraft I rather have it looked up by the post_options connector as it only happens one time and when needed and that way we don't need to update every single postBody when this value changes

Reply:
@mkraft let me done when this is removed so I can give it another look.

Reply:
It's pushed.